jsettlers.mapcreator/src/jsettlers/exceptionhandler/ErrorIcon.java

```diff
@@ -0,0 +1,58 @@
+package jsettlers.exceptionhandler;
```

Comment:
I created a new java project with helper scripts, one to install the copyright header on all files.
done